.github/CODEOWNERS

```
@@ -1 +1,2 @@
+.github/CODEOWNERS @SamMousa @brunobowden @advayDev1 @hspinks
 .github/workflows @SamMousa @brunobowden
```

```suggestion
/.github/workflows @SamMousa @brunobowden
```

Technically the `/` should be there